### ISSUE TITLE
Documentation  point to lore templates that don't exist

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#4682]
|[<img alt="lvh's avatar" src="https://avatars.githubusercontent.com/u/97816?s=50" width="50" height="50">](https://github.com/lvh)| @lvh reported|
|-|-|
|Trac ID|trac#4682|
|Type|defect|
|Created|2010-10-11 21:47:37Z|

```
lvh@bruichladdich:~/Development/twisted/twisted$ rgrep doc/howto/template.tpl . | grep -v historic
./doc/lore/howto/lore.xhtml:% lore --config template=doc/howto/template.tpl doc/howto/*.xhtml
./doc/core/development/policy/svn-dev.xhtml:./bin/lore/lore -p --config template=doc/howto/template.tpl doc/howto/*.xhtml
./doc/core/development/policy/svn-dev.xhtml:./bin/lore/lore -p --config template=doc/howto/template.tpl \
```

doc/howto/template.tpl doesn't actually exist anymore, it's doc/core/howto/template.tpl now.

Attachments:

* [lore-docfix-4682.diff](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/7d5/7d586d0a419fe8bf44dbbfd72cb5dc31b109ff1a/3c453946f0617a55963236a4665c2aeb622a504b.diff) (3509 bytes) - added by lvh on 2010-10-11 23:13:46Z - 

<details><summary>Searchable metadata</summary>

```
trac-id__4682 4682
type__defect defect
reporter__lvh lvh
priority__normal normal
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__lore lore
keywords__documentation_easy documentation easy
time__1286833657000000 1286833657000000
changetime__1300074021000000 1300074021000000
version__None None
owner__lvh lvh
cc__jesstess
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="lvh's avatar" src="https://avatars.githubusercontent.com/u/97816?s=50" width="50" height="50">](https://github.com/lvh)|@lvh set owner to @lvh|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="lvh's avatar" src="https://avatars.githubusercontent.com/u/97816?s=50" width="50" height="50">](https://github.com/lvh)<a name="note_2"></a>|@lvh commented|
|-|-|

I found some more places with this broken reference, I'll fix them while I'm at it:

```
lvh@bruichladdich:~/Development/twisted/lore-docfix-4682$ rgrep doc/howto . | grep -v historic | grep -v ChangeLog.Old
./doc/lore/howto/lore.xhtml:% lore --config template=doc/core/howto/template.tpl doc/howto/*.xhtml
./twisted/python/usage.py:or doc/howto/options.html in your Twisted directory.
./twisted/python/usage.py:    or doc/howto/options.html in your Twisted directory.
./twisted/internet/defer.py:    For more information about Deferreds, see doc/howto/core/defer.html or
./twisted/web/sux.py:    # fn = "/home/glyph/Projects/Twisted/doc/howto/ipc10paper.html"
./emacs/twisted-dev.el:  (with-cd (format "%s/doc/howto" twisted-dev-directory)
```

**Comment by twisted-trac** [comment]
|[<img alt="lvh's avatar" src="https://avatars.githubusercontent.com/u/97816?s=50" width="50" height="50">](https://github.com/lvh)<a name="note_3"></a>|@lvh removed owner|
|-|-|

When I run that command now, I get:

```
lvh@bruichladdich:~/Development/twisted/lore-docfix-4682$ rgrep doc/howto . | grep -v historic | grep -v ChangeLog.Old
./twisted/web/sux.py:    # fn = "/home/glyph/Projects/Twisted/doc/howto/ipc10paper.html"
```

But t.w.sux deserves its own ticket for the nasty things in its `if __name__ == "__main__":`.

Diff attached. Individual commits can be found at lp:~lvh/twisted/lore-docfix-4682

**Comment by twisted-trac** [comment]
|[<img alt="jesstess's avatar" src="https://avatars.githubusercontent.com/u/188336?s=50" width="50" height="50">](https://github.com/jesstess)<a name="note_4"></a>|@jesstess set owner to @lvh|
|-|-|

What a great catch. Thanks for fixing these.

In

```
-    For more information about Deferreds, see doc/howto/core/defer.html or
+    For more information about Deferreds, see doc/core/howto/defer.html or
```

might as well switch to the xhtml reference like you do elsewhere.

Can you also bump the copyright on twisted/python/usage.py.

Other than that, looks great, please merge!

**Comment by twisted-trac** [comment]
|[<img alt="lvh's avatar" src="https://avatars.githubusercontent.com/u/97816?s=50" width="50" height="50">](https://github.com/lvh)<a name="note_5"></a>|@lvh set status to `closed`|
|-|-|

(In [30997]) Merge lore-docfix-4682: fix links to lore templates that don't exist.


Author: lvh
Reviewer: jesstess
Fixes: [#4682](https://github.com/twisted/twisted/issues/4682)

There were some references to doc/howto/template.tpl, which doesn't actually exist anymore, it's doc/core/howto/template.tpl now. There was an additional small link fix in the Lore docs.
